JustLog/Classes/Logger.swift

```diff
@@ -118,6 +123,15 @@ public final class Logger: NSObject {
             
             timer?.run()
         }
+        
+        // custom logging
+        if enableCustomLogging, let customLogSender = customLogSender {
+            let customDestination = CustomDestination(sender: customLogSender)
+            // always send all logs to custom destination
+            customDestination.minLevel = .verbose
```

Comment:
Wouldn't it better to have `minLevel` as part of `CustomDestinationSender` protocol?

Reply:
Again, something for the future? These destinations are exposed in the API, so clients can update the `minLogLevel` as they like.

Reply:
Also, the protocol is purely for the sending part. Not the destination.